_field-types/mapping-parameters/index-parameter.md

```diff
@@ -173,3 +196,47 @@ The following error response indicates that the search query failed because the
   "status": 400
 }
 ```
+
+For text fields, the index parameter being false is enough to disable searching on that field (this is because text fields do not support doc_values), however, for the other field types, we must also disable `doc_values` to have the same effect.  
```

Comment:
```suggestion
For `text` fields, setting the `index` parameter to `false` disables searching on the field because `text` fields do not support `doc_values`. To make other fields not searchable, you must additionally set `doc_values` to  `false`.  
```